dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/GeneralTestDataProvider.kt

```diff
@@ -28,6 +28,23 @@ class GeneralTestDataProvider {
         )
     }
 
+    fun generateCompanyInformationWithNameAndIdentifiers(
+        lei: String?,
+        isin: List<String>?,
+        permId: String?,
+    ): CompanyInformation {
+        var identifiers = emptyMap<String, List<String>>()
+        if (!lei.isNullOrEmpty()) identifiers = identifiers + mapOf(IdentifierType.lei.value to listOf(lei))
+        if (!isin.isNullOrEmpty()) identifiers = identifiers + mapOf(IdentifierType.isin.value to isin)
+        if (!permId.isNullOrEmpty()) identifiers = identifiers + mapOf(IdentifierType.permId.value to listOf(permId))
+        return CompanyInformation(
+            "DummyCompany",
```

Comment:
Are those hardcoded values desired?

Reply:
@PaulChr99

Reply:
What happens if I upload a company with the same name twice (or multiple times) but with different identifiers? Do we interpret the next uploads as different companies or are we extending the identifiers list in the already existing company? If the former is true (new company every time), then this works as expected, since we want to operate on different company ids. If not (company information extended instead), we would need to consider making the company name a parameter. Concerning the headquarters and the country code - they are only provided with dummy information for being required fields

Reply:
I think if the identifiers are different then a new company entry will be created

Reply:
Yes, I just verified that. When the name remains the same but the identifiers change, then a different company with new companyId is created. In case that you enter an already used identifier you receive an error. Hence, this approach with a dummy company name should be fine, as it does not matter anyways (only companyId is important)

Reply:
ok